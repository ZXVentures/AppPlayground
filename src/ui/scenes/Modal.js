import React, { Component } from 'react' // eslint-disable-line no-unused-vars
import {
  StyleSheet,
  View
} from 'react-native'

export default class Modal extends Component {

  render(): React$Element<any> {
    return (
      <View style={styles.container}></View>
    )
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
    backgroundColor: '#0091ea'
  }
})
