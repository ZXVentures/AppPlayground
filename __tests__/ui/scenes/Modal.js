import 'react-native'
import React from 'react' // eslint-disable-line no-unused-vars
import Modal from 'ui/scenes/Modal'

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer'

describe('ui/scenes/Modal', () => {
  it('snapshot test', () => {
    const tree = renderer.create(
      <Modal />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
