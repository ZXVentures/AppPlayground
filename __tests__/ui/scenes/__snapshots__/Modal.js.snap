exports[`ui/scenes/Modal snapshot test 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#0091ea",
      "flex": 1,
      "justifyContent": "center",
    }
  } />
`;
